src/Oci8/Oci8Connection.php

```diff
@@ -216,9 +216,10 @@ protected function getDoctrineDriver()
      * @param string $sql (mixed)
      * @param array $bindings (kvp array)
      * @param int $returnType (PDO::PARAM_*)
+     * @param int $length
      * @return mixed $returnType
      */
-    public function executeFunction($sql, array $bindings = [], $returnType = PDO::PARAM_STR)
+    public function executeFunction($sql, array $bindings = [], $returnType = PDO::PARAM_STR, $length = null)
```

Comment:
Thanks for the updates. Just realized, the default [max length](http://php.net/manual/en/function.oci-bind-by-name.php) is `-1`. Do you think it's better if set `$length = -1` or `null` is ok?